unity/Holo/Assets/Scripts/ModelsCollection.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.IO;
+using System.Collections.Generic;
+
+using UnityEngine;
+
+/* Add this to any GameObject in a scene, to initialize and keep updating available model bundles. */
+public class ModelsCollection : MonoBehaviour
+{
+    private static ModelsCollection singleton;
+    public static ModelsCollection Singleton
+    {
+        get { return singleton; }
+    }
+
+    /* Absolute filenames to asset bundles with models. */
+    private string[] bundlesFiles;
+    private AssetBundleLoader[] bundles;
+
+    /* Suffix to recognize bundle filename. May be an extension (with dot) or a normal filename suffix. */
+    private const string bundleFileSuffix = "_bundle";
+
+    /* Initialize bundlesFiles */
+    private void Start()
+    {
+        singleton = this;
+        LoadBundlesFiles();
+    }
+
+    private void LoadBundlesFiles()
+    {
+        bundlesFiles = new string[] { };
+        LocalConfig localConfig = Resources.Load<LocalConfig>("LocalConfig");
+        if (localConfig != null && !string.IsNullOrEmpty(localConfig.BundlesDirectory))

```

Comment:
I'd rather go other way round with `if`. Check for error and use early return. Current main flow should not be under if then

Reply:
👍